Hunting Queries/AzureDiagnostics/CriticalPortsOpened.yaml

```diff
@@ -0,0 +1,38 @@
+id: 66d494c0-233c-438a-9b1b-5fe839790d38
+name: Check critical ports opened to the entire internet
+description: |
+  'Discover all critical ports from a list having rules like 'Any' for sourceIp, which means that they are opened to everyone. Critial ports shouldn't be opened to everyone, and should be filtered.'
```

Comment:
Because there is an extra single quote in "shouldn't", the format is incorrect when being used in Log Analytics directly.  Change "shouldn't" to "should not" and then this issue will not occur.  The YAML validator does not catch this because it is valid YAML, just for when using directly in Log Analytics. :)

Reply:
done